uint8 status
uint8 sensor_limitAhead
uint8 sensor_limitBehind
uint8 sensor_checkRack